[CI] NoMasterNodeIT.testNoMasterActions failed

NoMasterNodeIT.testNoMasterActions failed in 
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+multijob-unix-compatibility/os=opensuse/386/console

Could not reproduce locally with following:
```
./gradlew :server:integTest --tests "org.elasticsearch.cluster.NoMasterNodeIT.testNoMasterActions" \
  -Dtests.seed=F52513BEB5BB0A48 \
  -Dtests.security.manager=true \
  -Dtests.locale=so-DJ \
  -Dtests.timezone=Asia/Jakarta \
  -Dcompiler.java=12 \
  -Druntime.java=11
```

```
14:09:09 
14:09:09 
14:09:09 org.elasticsearch.cluster.NoMasterNodeIT > testNoMasterActions FAILED
14:09:09     java.lang.AssertionError: AcknowledgedResponse failed - not acked
14:09:09     Expected: <true>
14:09:09          but: was <false>
```
[consoleText.txt](https://github.com/elastic/elasticsearch/files/3164886/consoleText.txt)

--- a/server/src/test/java/org/elasticsearch/cluster/NoMasterNodeIT.java
+++ b/server/src/test/java/org/elasticsearch/cluster/NoMasterNodeIT.java
@@ -163,9 +163,7 @@ public class NoMasterNodeIT extends ESIntegTestCase {
         bulkRequestBuilder.setTimeout(timeout);
         checkWriteAction(bulkRequestBuilder);
 
-        disruptionScheme.stopDisrupting();
-
-        client().admin().cluster().prepareHealth().setWaitForGreenStatus().setWaitForNodes("3").execute().actionGet();
+        internalCluster().clearDisruptionScheme(true);
     }
 
     void checkUpdateAction(boolean autoCreateIndex, TimeValue timeout, ActionRequestBuilder<?, ?> builder) {

